config/locales/ar.yml

```diff
@@ -631,6 +631,7 @@ ar:
     engage_with_gov:
     fcdo_bloggers:
     follow:
+    follow_links:
```

Comment:
Do you know what the next steps are in terms of getting this key translated?

Reply:
Nope, unfortunately! Currently there is no contingency plan for importing translations. The mission got cut short 😢  I'm hoping the next team that inherits this work (Find and view which I'm moving to) can prioritise it, but we'll see.